template/_package.json

```diff
@@ -119,6 +119,9 @@
     <%_ if (ui === 'tachyons') { _%>
     "tachyons": "^4.11.1",
     <%_ } _%>
+    <%_ if (styleResources) { _%>
+    "@nuxtjs/style-resources": "^1.0.0",
```

Comment:
Can we put the package as alphabetical order ?

Reply:
Sorted the packages in alpha order and moved the conditional for tachyons up as part of the rest of the of 'ui ===' group.